js/components/tabs.js

```diff
@@ -27,6 +27,25 @@ class Tabs extends ImmutableComponent {
     this.onNextPage = this.onNextPage.bind(this)
     this.onNewTabLongPress = this.onNewTabLongPress.bind(this)
     this.wasNewTabClicked = this.wasNewTabClicked.bind(this)
+    this.onMouseLeave = this.onMouseLeave.bind(this)
+    this.onFrameClose = this.onFrameClose.bind(this)
+
+    this.state = {
+      fixTabWidth: null
+    }
+  }
+
+  onMouseLeave () {
+    this.setState({
+      fixTabWidth: null
+    })
+    this.forceUpdate()
```

Comment:
@bsclifton I do understand your approach. I dint do that since this tab width change s such a small state change. It has no impact on the other areas of the application.

Will make the change as you suggested.